tests/unit/client_test.py

```diff
@@ -0,0 +1,26 @@
+import os
+from docker.client import Client
+from .. import base
+
+TEST_CERT_DIR = os.path.join(
+    os.path.dirname(__file__),
+    'testdata/certs',
+)
+
+
+class ClientTest(base.BaseTestCase):
+    def setUp(self):
+        self.os_environ = os.environ.copy()
+
+    def tearDown(self):
+        os.environ = self.os_environ
+
+    def test_from_env(self):
```

Comment:
I felt it was more suitable to associate it with `Client` because it was testing a code path in there.